sample-shrinker.sh

```diff
@@ -251,6 +255,32 @@ prep_bitdepth_convert()
 }
 
 
+# Determine what do to about sample rate
+# - updates $change_summary with a text summary
+# - updates $*_args with sox args
+prep_samplerate_convert()
+{
+  # Prepare samplerate conversion
+  if [[ $samplerate -ne $target_samplerate ]]; then
+    change_summary+="${samplerate}      "
+    if [[ $samplerate -gt $target_samplerate ]]; then
+      change_summary+="${samplerate}->${target_samplerate}"
+      dst_args+=(--rate="$target_samplerate")
+
+    # Raise below-minimum samplerate samples to minimum samplerate (default: 11025)
+    elif (( samplerate < target_samplerate && samplerate < 11025 )); then
+      if [[ -n $minimum_samplerate ]]; then
+        dst_args+=(--rate="$minimum_samplerate")
+
+        change_summary+="${samplerate}->$minimum_samplerate+M"
```

Comment:
This is the same symbol as a minimum bit depth change gets... how should we distinguish them?  Maybe a two-character code, like `+Mr` and `+Md`?  :thinking:

Reply:
Wasn't sure what the M was for :] - it just means 'modified' or what?

Reply:
In my head, `+` meant "modified" and `M` meant "**M**inumum," but its really just an arbitrary code to denote what will/is being changed.

Reply:
How do you trigger / see this anyway?
Happy to leave it or set it to anything you'd like - just let me know.